### PR TITLE
Fix #181

### LINKED ISSUE #181: Landing page Portfolio: Background does not cover all portfolio
- Background does not cover all portfolio
- Background should not have opacity 50%
Screenshot 2023-07-24 at 10 26 38 AM

### PR DESCRIPTION
**Pull request by LeonardoLopesHonda**
## Issue

Fix #181 by changing 'picture' element properties (image is stretched with this change).

## 🔗 Test URLs:

- Before: https://main--sgws-creative--leonardolopeshonda.hlx.page
- After: https://issue-181--sgws-creative--leonardolopeshonda.hlx.page

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -435,6 +440,11 @@ main .section.background-image:not(:first-child) > picture {
   top: auto;
 }
 
+main:not(.product) .section.background-image:not(:first-child) > picture {
```
Comment (tmathern) [inline styles/styles.css:443]:
```suggestion
body:not(.product) .section.background-image:not(:first-child) > picture {
```

**Review by tmathern — CHANGES_REQUESTED** [review]
Product pages are still broken. 

Test pages: 
Works:
- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1
- After: https://test-issue-181-231--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1
Breaks:
- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/2
- After: https://test-issue-181-231--sgws-creative--hlxsites.hlx.page/bloominbrands/p/2
- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/4
- After: https://test-issue-181-231--sgws-creative--hlxsites.hlx.page/bloominbrands/p/4
- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/5
- After: https://test-issue-181-231--sgws-creative--hlxsites.hlx.page/bloominbrands/p/5
- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/7
- After: https://test-issue-181-231--sgws-creative--hlxsites.hlx.page/bloominbrands/p/7


See code suggestion.

File: styles/styles.css (hunk 1 above)
Comment (LeonardoLopesHonda) [reply styles/styles.css:443]: your suggestion solved all the problems.

File: styles/styles.css [hunk 2]
```diff
@@ -135,12 +135,16 @@ main div.section.images-border .default-content-wrapper h2::after {
 .product .section:not(.quote) .default-content-wrapper h2::after {
   content: '';
   display: block;
-  border-bottom: 3px solid var(--secondary-text-color-background);
+  border-bottom: 3px solid var(--primary-color);
```
Comment (allentsliu) [inline styles/styles.css:138]: is this added by accident?

File: styles/styles.css [hunk 3]
```diff
@@ -135,12 +135,16 @@ main div.section.images-border .default-content-wrapper h2::after {
...
   padding-top: 0.5vw;
   width: 12.5%;
   margin: 1px auto;
 }
 
+.product .section:not(.quote.neutral) .default-content-wrapper h2::after {
+  border-bottom: 3px solid var(--neutral-color);
+}
+
```
Comment (allentsliu) [inline styles/styles.css:147]: Is this added by accident?

Comment (LeonardoLopesHonda) [reply styles/styles.css:147]: That part was changed after I used `git pull`

File: styles/styles.css (hunk 2 above)
Comment (tmathern) [reply styles/styles.css:138]: @allentsliu Please make sure it doesn't undo recent fixes.

File: styles/styles.css (hunk 3 above)
Comment (tmathern) [reply styles/styles.css:147]: @allentsliu Please make sure it doesn't undo recent fixes.

File: styles/styles.css (hunk 2 above)
Comment (allentsliu) [reply styles/styles.css:138]: @LeonardoLopesHonda  could you revert it if it is not part of your change? Thanks!

File: styles/styles.css (hunk 3 above)
Comment (allentsliu) [reply styles/styles.css:147]: @LeonardoLopesHonda  could you remove it if it is not part of your change? Thanks!

**Review by allentsliu — APPROVED** [review]

**Review by tmathern — APPROVED** [review]

**Comment by tmathern** [comment]
LGTM!

**Pull request merged**